src/Components/Home.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+
+import Navbar from './Navbar'
+import GetIcons from './Icons/GetIcons'
+
+function Home() {
+  return (
+    <>
+      <Navbar
+        start={
+          <Link to="/search" aria-label="Search">
```

Comment:
Because the search icon is a link it turns blue once visited. Is that something we want? 

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question&nbsp; | &nbsp;**Tags:** Inefficient

Reply:
I kept this the same as it was before (similar to the back button on the profile page)